DATA/SHIPS/loadouts_li.ini

```diff
@@ -1217,7 +1217,7 @@ cargo = ge_s_battery_01, 8
 [Loadout]
 nickname = li_n_li_elite2_loadout_d6_b
 equip = li_engine_fighter_heavy01, HpEngine01
-equip = cv_shield_neutrino_heavy_opt01, HpShield02
+equip = ku_shield_neutrino_heavy_opt01, HpShield02

```

Comment:
Neutrino shield is wrong here, replace with liberty graviton please

Reply:
Fixed in the next batch of changes